scripts/close-milestone.sh

```diff
@@ -0,0 +1,8 @@
+echo "'.[] | select(.title | contains("$1"))'" | read jqMilestoneSearchCommand
```

Comment:
Whoops shebang + `set -e -o pipefail -u`, as in the other scripts (https://virtuslab.github.io/tips/#shell/strict-mode)

Reply:
Also, `read -r`

Reply:
Or just `foo=$(...)` instead of `... | read foo`, more natural

Reply:
Thank you for this. read was as well removing the whitespace which was causing error in jq. Let me check and see.